### ISSUE TITLE
Why is the theme have sawtooth?

### ISSUE THREAD

**Issue by kgggggb** [c0re100/qBittorrent-Enhanced-Edition#405]
### qBittorrent & operating system versions

qBittorrent Enhanced: 4.4.3.12 x64 .AppImage
OS: Ubuntu 20.04

### What is the problem?

Strange sawtooth.
Original : [Imagur](https://i.imgur.com/JxM2WOq.png)
qBittorrent Enhanced: [1](https://i.imgur.com/o2VeWLi.png) [2](https://i.imgur.com/fAlD2ee.png)

**Comment by abcfy2** [comment]
Did you mean the dotted box ?

I thinks it's only mean you've clicked or selected this line or item. When you clicked outside, it should disappear.

I think it's GTK2 theme designed by themself.

**Comment by kgggggb** [comment]
> Did you mean the dotted box ?
> 
> I thinks it's only mean you've clicked or selected this line or item. When you clicked outside, it should disappear.
> 
> I think it's GTK2 theme designed by themself.

Yes, the problem is the original one doesn't have it, I am thinking is there a way to get rid of it.
